test/Simps.lean

```diff
@@ -123,6 +123,7 @@ example (n : ℕ) : foo.rfl.toFun n = n := by rw [foo.rfl_toFun, id]
 example (n : ℕ) : foo.rfl.invFun n = n := by rw [foo.rfl_invFun]
 
 /- the declarations are `simp` lemmas -/
+set_option linter.dupNamespace false in
```

Comment:
Why not change `foo` to `bar` and make the linter happy?

Reply:
Thanks for the suggestion: I changed the names and removed the `set_option`.